tracker-main/helper.py

```diff
@@ -3,10 +3,11 @@
 import requests
 import time
 import re
-from web3 import Web3
```

Comment:
extra web3 import.

It doesnt seems like we would need web3 at all for main.py